dana-bot/src/bot.js

```diff
@@ -1025,6 +1025,8 @@ function userRunEnd(msg) {
   if (!fs.existsSync(f))
     fs.mkdirSync(f);
 
+  globalBot.currentRun.exportedTo = [];
```

Comment:
I wonder if we should prefer something more generic here. There could have different post-build processes that wish to store metadata too. 

WDYT of something like 'globalBot.currentRun.metadata = {}' where 'metadata' is an object (not an array) where each key matches a post-build process' metadata object.

For example, the data export could be stored as globalBot.currentRun.metadata.exportedTo